lib/src/components/NxTransferListHalf/__tests__/NxTransferListHalf.test.tsx

```diff
@@ -171,9 +168,23 @@ describe('NxTransferListHalf', function() {
     expect(otherItem).toHaveText('baz');
   });
 
+  it('does not render move icon and checkbox input when onItemChange is not provided', function() {
+    const items = [{
+          id: 1,
+          displayName: 'foo'
+        }],
+        componentDisableItemMove = getMounted({ items }),
+        select = componentDisableItemMove
+            .find('.nx-transfer-list__item-list .nx-transfer-list__item .nx-transfer-list__select');
+
+    expect(select.find(NxFontAwesomeIcon)).not.toExist();
+    expect(select.find('input.nx-transfer-list__checkbox')).not.toExist();
+  });
+
   it('sets the item checked prop to isSelected', function() {
-    const withSelected = getMounted({ items: [{ id: 1, displayName: 'foo' }], isSelected: true }),
-        withoutSelected = getMounted({ items: [{ id: 1, displayName: 'foo' }] });
+    const onItemChange = jest.fn(),
+        withSelected = getMounted({ items: [{ id: 1, displayName: 'foo' }], isSelected: true, onItemChange }),
+        withoutSelected = getMounted({ items: [{ id: 1, displayName: 'foo' }], onItemChange });
```

Comment:
Should test `isSelected = false` as well

Reply:
Updated here 4dc735f5e5008c35a9245561f423c697ce343183.